CommonInstall/util_init.pl

```diff
@@ -324,8 +327,10 @@ ()
 		$CUR_DISTRO_VENDOR = "redhat";
 	} elsif ( -s "/etc/almalinux-release" ) {
 		$CUR_DISTRO_VENDOR = "redhat";
-	} elsif ( -s "/etc/circle-releas" ) {
+	} elsif ( -s "/etc/circle-release" ) {
 		$CUR_DISTRO_VENDOR = "redhat";
+	} elsif ( -s "/etc/opencloudos-release" ) {
```

Comment:
If you want to enable for the stream release, then I think you should check for the `/etc/opencloudos-stream-release` file.